README.md

```
@@ -640,6 +640,16 @@ Image processing with OpenCV works on the client side.
 
 </details>
 
+## Sponsors
+
+### Streamlit (Snowflake)
+
+[<img src="https://streamlit.io/images/brand/streamlit-mark-color.png" height="100" />](https://streamlit.io/) [<img alt="Snowflake" src="https://docs.snowflake.com/images/favicon/apple-touch-icon.png" height="100" />](https://www.snowflake.com/)
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)